src/plugins/auto_batch/src/compiled_model.cpp

```diff
@@ -154,93 +129,103 @@ std::pair<CompiledModel::WorkerInferRequest&, int> CompiledModel::GetWorkerInfer
+                                            ov::execution_devices.name()};
+        } else if (name == METRIC_KEY(SUPPORTED_CONFIG_KEYS)) {
+            return std::vector<std::string>{ov::auto_batch_timeout.name()};
+        } else if (name == ov::execution_devices) {
+            return m_compiled_model_without_batch->get_property(name);
+        } else if (name == ov::loaded_from_cache) {
+            return m_compiled_model_without_batch->get_property(ov::loaded_from_cache.name());
+        } else {
+            OPENVINO_THROW("Unsupported Compiled Model Property: ", name);
         }
-        IE_THROW(NotFound) << name << " not found in the ExecutableNetwork config";
     }
+    OPENVINO_SUPPRESS_DEPRECATED_END
 }
 
-InferenceEngine::Parameter CompiledModel::GetMetric(const std::string& name) const {
-    if (name == METRIC_KEY(OPTIMAL_NUMBER_OF_INFER_REQUESTS)) {
-        auto reqs = 0;
-        try {
-            auto hint = m_model_without_batch->GetConfig(CONFIG_KEY(PERFORMANCE_HINT_NUM_REQUESTS)).as<std::string>();
-            reqs = InferenceEngine::PerfHintsConfig::CheckPerformanceHintRequestValue(hint);
-            if (!reqs)  // no limitations from user, let's deduce the full blown #requests
-                // (multiplied by the devices capabilities to run multiple <batched> requests for further perf)
-                reqs =
-                    m_device_info.batch_for_device *
-                    m_model_without_batch->GetMetric(METRIC_KEY(OPTIMAL_NUMBER_OF_INFER_REQUESTS)).as<unsigned int>();
-        } catch (const InferenceEngine::Exception&) {
-        }
-        reqs = std::max(reqs, m_device_info.batch_for_device);  // round up to the possible  user's value
-        IE_SET_METRIC_RETURN(OPTIMAL_NUMBER_OF_INFER_REQUESTS, reqs);
-    } else if (name == METRIC_KEY(NETWORK_NAME)) {
-        IE_SET_METRIC_RETURN(NETWORK_NAME,
-                             m_model_without_batch->GetMetric(METRIC_KEY(NETWORK_NAME)).as<std::string>());
-    } else if (name == METRIC_KEY(SUPPORTED_METRICS)) {
-        IE_SET_METRIC_RETURN(SUPPORTED_METRICS,
-                             {METRIC_KEY(OPTIMAL_NUMBER_OF_INFER_REQUESTS),
-                              METRIC_KEY(SUPPORTED_METRICS),
-                              METRIC_KEY(NETWORK_NAME),
-                              METRIC_KEY(SUPPORTED_CONFIG_KEYS),
-                              ov::execution_devices.name()});
-    } else if (name == METRIC_KEY(SUPPORTED_CONFIG_KEYS)) {
-        IE_SET_METRIC_RETURN(SUPPORTED_CONFIG_KEYS,
-                             {CONFIG_KEY(AUTO_BATCH_TIMEOUT)});  // only timeout can be changed on the fly
-    } else if (name == ov::execution_devices) {
-        return m_model_without_batch->GetMetric(name);
-    } else {
-        IE_THROW() << "Unsupported Network metric: " << name;
-    }
+void CompiledModel::export_model(std::ostream& model) const {
+    OPENVINO_NOT_IMPLEMENTED;
```

Comment:
```suggestion
        OPENVINO_ASSERT_HELPER(::ov::NotImplemented,
                               "",
                               false,
                               "Not Implemented",
                               "CompiledModel::export_model is not supported by this auto_batch plugin!");
```

Will be better to provide more detail logs?

Reply:
keep the style with other plugin for now. Next PR can for this